Dockerfile.builder.multi_arch

```diff
@@ -0,0 +1,66 @@
+#
+# Builder dependencies. This takes a long time to build from scratch!
+# Also note that if build fails due to C++ internal error or similar,
+# it is possible that the image build needs more RAM than available by
+# default on non-Linux docker installs.
+#
+# Using cilium-builder as the base to ensure libc etc. are in sync.
+#
+# The base image here should be multi-arched beforehand when used for non-amd64
+# platforms, e.g, arm64 platform.
+#
+FROM quay.io/cilium/cilium-builder:2020-05-05 as builder
+
+LABEL maintainer="maintainer@cilium.io"
+
+ARG ARCH=amd64
+
+WORKDIR /go/src/github.com/cilium/cilium/envoy
+COPY . ./
+
+#
+# Additional Envoy Build dependencies
+#
+RUN apt-get update \
+    && DEBIAN_FRONTEND=noninteractive apt-get upgrade -y --no-install-recommends \
+	&& DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+		automake \
+		cmake \
+		g++ \
+		git \
+		libtool \
+		make \
+		ninja-build \
+		python3 \
+		python \
+		wget \
+		zip \
+	&& apt-get clean \
+	&& rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+#
+# Install Bazel
+#
+RUN export BAZEL_VERSION=`cat .bazelversion` \
+        && if [ "$ARCH" = "amd64" ] ; then \
+	     curl -sfL https://github.com/bazelbuild/bazel/releases/download/${BAZEL_VERSION}/bazel-${BAZEL_VERSION}-installer-linux-x86_64.sh -o bazel-${BAZEL_VERSION}-installer-linux-x86_64.sh \
+	     && chmod +x bazel-${BAZEL_VERSION}-installer-linux-x86_64.sh \
+	     && ./bazel-${BAZEL_VERSION}-installer-linux-x86_64.sh \
+	     && mv /usr/local/bin/bazel /usr/bin \
+	     && rm bazel-${BAZEL_VERSION}-installer-linux-x86_64.sh ; \
+           elif [ "$ARCH" = "arm64" ] ; then \
+             apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y curl unzip zip build-essential openjdk-11-jdk && \
```

Comment:
`bazel` packages on OBS are available for x86_64/amd64 too. Maybe let's install them for every architecture and not use the official script? What do you think?